nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -237,26 +236,25 @@ private Handler loadInitialWars(final Set<Bundle> bundles) {
                 webContentViewerWar = war;
             } else if (war.getName().toLowerCase().startsWith("nifi-web")) {
                 webUiWar = war;
+            } else if (war.getName().toLowerCase().startsWith("nifi-standard-content")) {
```

Comment:
NiFi also ships with content viewers in other NARs. Specifically, I believe we have some for viewing images in the media NAR. I'm not sure we can generally consider stuff in `nifi-standard-content` as the only standard content viewers.

Reply:
Are you suggesting we name `nifi-standard-content-viewer` something a bit more specific, like `nifi-standard-file-content-viewer`?

Reply:
@mcgilman At this time, there is no media nar in the nifi distribution so I think that is the cause for the disconnect here...
The last version it existed was 1.9.2. 

This is likely a bug so I opened https://issues.apache.org/jira/browse/NIFI-7931 for it.

But, yes, this will need to be accounted for as well...